template/src/client/App.js

```diff
@@ -1,6 +1,5 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
-import './App.css';
```

Comment:
Can be solved with `new webpack.NormalModuleReplacementPlugin(/\.(css|scss)$/, 'node-noop')`